depthai_demo.py

```diff
@@ -171,7 +218,35 @@ def get_label_text(self, label):
             return self.labels[int(label)]
         else:
             print(f"Label of ouf bounds (label_index: {label}, available_labels: {len(self.labels)}")
-            return label
+            return str(label)
+    
+    def draw_detections(self, source, detections):
+        def draw_detection(frame, detection):
+            bbox = frame_norm(frame, [detection.xmin, detection.ymin, detection.xmax, detection.ymax])
+            cv2.rectangle(frame, (bbox[0], bbox[1]), (bbox[2], bbox[3]), (255, 0, 0), 2)
+            cv2.putText(frame, self.get_label_text(detection.label), (bbox[0] + 10, bbox[1] + 20),
+                        cv2.FONT_HERSHEY_TRIPLEX, 0.5, 255)
```

Comment:
similarly, add `text_type` and use that instead  cv2.FONT_HERSHEY_TRIPLEX across the project, so it's configurable.